internal/api/utils.go

```diff
@@ -265,56 +260,109 @@ func TransformComponentUnits(jsonData datatypes.JSON) map[string]interface{} {
 			if math.Abs(cpuInCores) < 1 {
 				cpuInMillicores := cpuInCores * 1000
 				cpu["amount"] = math.Round(cpuInMillicores) // millicore values are rounded & don't require decimal precision
-				cpu["format"] = "millicores"
+				cpu["format"] = "m"
 			} else {
 				cpu["amount"] = truncateToThreeDecimalPlaces(cpuInCores)
-				cpu["format"] = "cores"
+				cpu["format"] = nil
 			}
 		}
 		return nil
 	}
 
+	// Current
```

Comment:
Just //Current is not helping to understand the context, can you add more details?

Reply:
I have elaborated on the comment